src/app/public/modules/ag-grid/cell-renderers/cell-renderer-row-selector/cell-renderer-row-selector.component.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  ChangeDetectorRef,
+  ChangeDetectionStrategy,
+  Component,
+  OnInit
+} from '@angular/core';
+
+import {
+  SkyLibResourcesService
+} from '@skyux/i18n';
+
+import {
+  ICellRendererAngularComp
+} from 'ag-grid-angular';
+
+import {
+  ICellRendererParams,
+  RowNode,
+  RowSelectedEvent
+} from 'ag-grid-community';
+
+@Component({
+  selector: 'sky-ag-grid-cell-renderer-row-selector',
+  templateUrl: './cell-renderer-row-selector.component.html',
+  styleUrls: ['./cell-renderer-row-selector.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SkyAgGridCellRendererRowSelectorComponent implements ICellRendererAngularComp, OnInit {
+  public checked: boolean;
+  public rowNode: RowNode;
+  public checkboxLabel: string;
+  private params: ICellRendererParams;
+  private rowNumber: number;
+
+  constructor(
+    private changeDetection: ChangeDetectorRef,
+    private libResources: SkyLibResourcesService
+    ) { }
+
+  public agInit(params: ICellRendererParams): void {
+    this.params = params;
+    this.checked = this.params.value;
+    this.rowNode = this.params.node;
+    this.rowNumber = this.params.rowIndex + 1;
+
+    this.rowNode.addEventListener(RowNode.EVENT_ROW_SELECTED, (event: RowSelectedEvent) => { this.rowSelectedListener(event); });
+    this.rowNode.setSelected(this.checked);
+  }
+
+  public ngOnInit(): void {
+    this.libResources.getString('sky_ag_grid_row_selector_aria_label', this.rowNumber)
+    .subscribe(label => {
```

Comment:
All subscriptions need to be completed by the consuming component, in the `ngOnDestroy` lifecycle method.

For example: https://github.com/blackbaud/skyux-forms/blob/master/src/app/public/modules/toggle-switch/toggle-switch.component.ts#L122